app/services/search/query_builders/feed_content.rb

```diff
@@ -87,11 +87,10 @@ def add_sort
         @body[:sort] = { @params[:sort_by] => @params[:sort_direction] }
       end
 
+      # Highlights search results in HTML using `<mark>` returning 2 fragments on 75 chars each ordering them by score
```

Comment:
`pg_search` supports highlighting and can be configured (what tag to use, etc.). See: https://github.com/Casecommons/pg_search#highlight